src/main/java/uk/gov/ons/census/action/builders/PrintFileDtoBuilder.java

```diff
@@ -15,7 +15,9 @@
           ActionType.P_RL_1RL1A,
           ActionType.P_RL_1RL2BA,
           ActionType.P_RL_2RL1A,
-          ActionType.P_RL_2RL2BA
+          ActionType.P_RL_2RL2BA,
+          ActionType.P_RL_1IRL1,
+          ActionType.P_RL_1IRL2B
           // Adding to this list will result in no UAC being sent on the printed
           // letter/questionnaire. Please be certain of what you're doing before adding to it.
```

Comment:
@NickGrantONS Already done!

Reply:
ah dammit I searched for this and musta missed it